src/user_templates_api/templates/jupyter_lab/templates/hra_api_tutorial/template.txt

```diff
@@ -0,0 +1,1283 @@
+      "<pre>\n",
+      "Accepted Parameters:\n",
+      "    age - (MinMax) (optional) (min = 1.0; max = 110.0)\n",
+      "    bmi - (MinMax) (optional) (min = 13.0; max = 83.0)\n",
+      "    cache - (boolean)  (optional) (defaults to True)\n",
+      "    sex - (string)  (optional) (male, female, both)\n",
+      "    technologies - (string) (optional) (List of technologies - api_instance.technology_names)\n",
+      "    providers - (string) (optional) \n",
+      "    ontology_terms - (string) (optional) \n",
+      "    cell_type_terms - (string) (optional) \n",
+      "    spatial - SpatialSearch(x, y, z, radius, target) (optional) \n",
+      "    token - (string) (optional) \n",
+      "</pre>"
+    ]
+  },
+  {
+    "cell_type": "code",
+    "execution_count": null,
+    "id": "dabe4526",
+    "metadata": {},
+    "outputs": [],
+    "source": [
+      "age = MinMax(min=45.0, max=60.0)\n",
+      "bmi = MinMax(min=65.0, max=83.0)\n",
+      "cache = True\n",
+      "sex = \"female\"\n",
+      "technologies = []\n",
+      "providers = []\n",
+      "ontology_terms = [\"http://purl.obolibrary.org/obo/UBERON_0000955\",]\n",
+      "cell_type_terms = [\"http://purl.obolibrary.org/obo/CL_0000000\",] \n",
+      "spatial = [SpatialSearch(\n",
+      "            x=84.0,\n",
+      "            y=146.0,\n",
+      "            z=53.0,\n",
+      "            radius=12.0,\n",
+      "            target=\"http://purl.org/ccf/latest/ccf.owl#VHFAllenBrain\",\n",
+      "        ),]\n",
+      "token = \"\"\n",
+      "\n",
+      "try:\n",
+      "    api_response = api_instance.ontology_term_occurences(age=age, bmi=bmi, cache=cache, ontology_terms=ontology_terms, cell_type_terms=cell_type_terms, providers=providers, sex=sex, spatial=spatial, technologies=technologies, token=token)\n",
+      "    pprint(api_response)\n",
+      "except hra_api_client.ApiException as e:\n",
+      "    print(\"Exception when calling DefaultApi->aggregate_results: %s\\n\" % e)"
+    ]
+  },
+  {
+    "attachments": {},
+    "cell_type": "markdown",
+    "id": "ae1132cc",
+    "metadata": {},
+    "source": [
+      "***\n",
+      "<a id='cell_term_occ'></a>\n",
+      "#### Get number of cell type term occurrences for a search\n",
+      "<pre>\n",
+      "Accepted Parameters:\n",
+      "    age - (MinMax) (optional) (min = 1.0; max = 110.0)\n",
+      "    bmi - (MinMax) (optional) (min = 13.0; max = 83.0)\n",
+      "    cache - (boolean)  (optional) (defaults to True)\n",
+      "    sex - (string)  (optional) (male, female, both)\n",
+      "    technologies - (string) (optional) (List of technologies - api_instance.technology_names)\n",
+      "    providers - (string) (optional) \n",
+      "    ontology_terms - (string) (optional) \n",
+      "    cell_type_terms - (string) (optional) \n",
+      "    spatial - SpatialSearch(x, y, z, radius, target) (optional) \n",
+      "    token - (string) (optional) \n",
+      "</pre>"
+    ]
+  },
```

Comment:
This cell produces ~600 lines of output, that are all obolibrary links. For a tutorial, I think it might be more usable if it would show e.g. the first 10 outputs. If a user wants to see all, they could run this themselves.